src/CallbackList.php

```diff
@@ -1,22 +1,22 @@
-<?php
+<?php declare(strict_types = 1);
 
 namespace Sminnee\CallbackList;
 
+// @phpcs:disable SlevomatCodingStandard.TypeHints.DisallowMixedTypeHint.DisallowedMixedTypeHint
+
 /**
  * Manages a list of callbacks that can be called sequentially with {@link call()}
  */
 class CallbackList
 {
-    /**
-     * @var callable[]
-     */
+    /** @var array<callable> */
```

Comment:
We'll want to check that PHPStorm / Intellephense parse this syntax

Reply:
I was using PHPStorm :)

Reply:
Cool good enough for me; vs code matches that